src/test/debugger/android/androidPlatform.test.ts

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as Q from "q";
+import * as path from "path";
+import * as mockFs from "mock-fs";
+
+import {AndroidPlatform} from "../../../debugger/android/androidPlatform";
+import {IRunOptions} from "../../../common/launchArgs";
+import {FileSystem} from "../../../common/node/fileSystem";
+import {ReactNative022} from "../../../test/resources/reactNative022";
+import {DeviceHelper} from "../../../test/resources/simulators/deviceHelper";
+import {AVDManager} from "../../../test/resources/simulators/avdManager";
+import {FakeExtensionMessageSender} from "../../../test/resources/fakeExtensionMessageSender";
+import {ExtensionMessage} from "../../../common/extensionMessaging";
+import {RecordingsHelper} from "../../resources/recordingsHelper";
+import {RemoteExtension} from "../../../common/remoteExtension";
+
+import "should";
+
+// TODO: Launch the extension server
```

Comment:
This is a future improvement we should do when we have time and the investment makes sense. I don't think it's worth it to create a task at the moment.